integration-tests/tests/src/tests/queries.ts

```diff
@@ -76,8 +76,11 @@ describe("Realm Query Language", () => {
         t: new Date("2004-02-26T10:15:02"),
       });
     });
+    //@ts-expect-error Test about to change
```

Comment:
Could you fill me in on `Test about to change`? :)

Reply:
Yes haha, I just know this file will be completely revamped the moment we merge test-migrations so did not want to do anything about it, should have been clearer in my comment.